### ISSUE TITLE
Issue with new fix

### ISSUE THREAD

**Issue by pwalczysko** [ome/omero-guide-ilastik#35]
@jburel 

On branch https://github.com/jburel/omero-guide-ilastik/tree/build_issue the mamba env builds as expected.

But, on local start on Mac, I have on 
pixel_classification.ipynb
notebook

on the cell

```
Load each image as an 5D-numpy array and analyze.

Save the probabilities as an OMERO image
```

following long error (attached as text file because cannot be pasted as too long):

[error.txt](https://github.com/user-attachments/files/17436966/error.txt)

**Comment by pwalczysko** [comment]
The notebook

pixel_classification_zarr 

ended also with an error, also in the cell where the images are due to be analyzed by ilastik

```
INFO ilastik.app: config file location: <none>

INFO:ilastik.app:config file location: <none>

INFO ilastik.app: Starting ilastik from "/Users/pwalczysko/opt/anaconda3/envs/omero-guide-ilastik/ilastik-meta".

INFO:ilastik.app:Starting ilastik from "/Users/pwalczysko/opt/anaconda3/envs/omero-guide-ilastik/ilastik-meta".

Starting ilastik from "/Users/pwalczysko/opt/anaconda3/envs/omero-guide-ilastik/ilastik-meta".
INFO ilastik.app: Resetting lazyflow thread pool with 2 threads.

INFO:ilastik.app:Resetting lazyflow thread pool with 2 threads.

INFO ilastik.app: Configuring lazyflow RAM limit to 2.0GiB

INFO:ilastik.app:Configuring lazyflow RAM limit to 2.0GiB

INFO lazyflow.utility.memory: Available memory set to 2.0GiB

INFO:lazyflow.utility.memory:Available memory set to 2.0GiB
WARNING 2024-10-18 16:23:28,120 opConservationTracking 76570 8605590144 Could not find any ILP solver
WARNING:ilastik.applets.tracking.conservation.opConservationTracking:Could not find any ILP solver
WARNING 2024-10-18 16:23:28,131 opStructuredTracking 76570 8605590144 Could not find any ILP solver
WARNING:ilastik.applets.tracking.structured.opStructuredTracking:Could not find any ILP solver
WARNING 2024-10-18 16:23:28,133 structuredTrackingWorkflow 76570 8605590144 Could not find any learning solver. Tracking will use flow-based solver (DPCT). Learning for tracking will be disabled!
WARNING:ilastik.workflows.tracking.structured.structuredTrackingWorkflow:Could not find any learning solver. Tracking will use flow-based solver (DPCT). Learning for tracking will be disabled!
WARNING 2024-10-18 16:23:29,542 __init__ 76570 8605590144 Failed to import NeuralNet workflow; check dependencies: No module named 'tiktorch.launcher'
Traceback (most recent call last):
  File "/Users/pwalczysko/opt/anaconda3/envs/omero-guide-ilastik/ilastik-meta/ilastik/ilastik/workflows/__init__.py", line 137, in <module>
    from .nnClassification import NNClassificationWorkflow
  File "/Users/pwalczysko/opt/anaconda3/envs/omero-guide-ilastik/ilastik-meta/ilastik/ilastik/workflows/nnClassification/__init__.py", line 21, in <module>
    from .nnClassificationWorkflow import NNClassificationWorkflow
  File "/Users/pwalczysko/opt/anaconda3/envs/omero-guide-ilastik/ilastik-meta/ilastik/ilastik/workflows/nnClassification/nnClassificationWorkflow.py", line 29, in <module>
    from ilastik.applets.serverConfiguration import ServerConfigApplet
  File "/Users/pwalczysko/opt/anaconda3/envs/omero-guide-ilastik/ilastik-meta/ilastik/ilastik/applets/serverConfiguration/__init__.py", line 21, in <module>
    from .serverConfigApplet import ServerConfigApplet
  File "/Users/pwalczysko/opt/anaconda3/envs/omero-guide-ilastik/ilastik-meta/ilastik/ilastik/applets/serverConfiguration/serverConfigApplet.py", line 29, in <module>
    from lazyflow.operators import tiktorch
  File "/Users/pwalczysko/opt/anaconda3/envs/omero-guide-ilastik/ilastik-meta/ilastik/lazyflow/operators/tiktorch/__init__.py", line 3, in <module>
    from .classifier import TiktorchConnectionFactory
  File "/Users/pwalczysko/opt/anaconda3/envs/omero-guide-ilastik/ilastik-meta/ilastik/lazyflow/operators/tiktorch/classifier.py", line 41, in <module>
    from tiktorch.launcher import LocalServerLauncher, RemoteSSHServerLauncher, SSHCred, ConnConf
ModuleNotFoundError: No module named 'tiktorch.launcher'
WARNING:ilastik.workflows:Failed to import NeuralNet workflow; check dependencies: No module named 'tiktorch.launcher'
Traceback (most recent call last):
  File "/Users/pwalczysko/opt/anaconda3/envs/omero-guide-ilastik/ilastik-meta/ilastik/ilastik/workflows/__init__.py", line 137, in <module>
    from .nnClassification import NNClassificationWorkflow
  File "/Users/pwalczysko/opt/anaconda3/envs/omero-guide-ilastik/ilastik-meta/ilastik/ilastik/workflows/nnClassification/__init__.py", line 21, in <module>
    from .nnClassificationWorkflow import NNClassificationWorkflow
  File "/Users/pwalczysko/opt/anaconda3/envs/omero-guide-ilastik/ilastik-meta/ilastik/ilastik/workflows/nnClassification/nnClassificationWorkflow.py", line 29, in <module>
    from ilastik.applets.serverConfiguration import ServerConfigApplet
  File "/Users/pwalczysko/opt/anaconda3/envs/omero-guide-ilastik/ilastik-meta/ilastik/ilastik/applets/serverConfiguration/__init__.py", line 21, in <module>
    from .serverConfigApplet import ServerConfigApplet
  File "/Users/pwalczysko/opt/anaconda3/envs/omero-guide-ilastik/ilastik-meta/ilastik/ilastik/applets/serverConfiguration/serverConfigApplet.py", line 29, in <module>
    from lazyflow.operators import tiktorch
  File "/Users/pwalczysko/opt/anaconda3/envs/omero-guide-ilastik/ilastik-meta/ilastik/lazyflow/operators/tiktorch/__init__.py", line 3, in <module>
    from .classifier import TiktorchConnectionFactory
  File "/Users/pwalczysko/opt/anaconda3/envs/omero-guide-ilastik/ilastik-meta/ilastik/lazyflow/operators/tiktorch/classifier.py", line 41, in <module>
    from tiktorch.launcher import LocalServerLauncher, RemoteSSHServerLauncher, SSHCred, ConnConf
ModuleNotFoundError: No module named 'tiktorch.launcher'

INFO ilastik.shell.projectManager: Opening Project: /var/folders/t4/z2zdsn015673vzl4lw15_bkm0000gn/T/tmpjbc5pqok/pixel-class-wednesday.ilp

INFO:ilastik.shell.projectManager:Opening Project: /var/folders/t4/z2zdsn015673vzl4lw15_bkm0000gn/T/tmpjbc5pqok/pixel-class-wednesday.ilp

---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
~/opt/anaconda3/envs/omero-guide-ilastik/lib/python3.7/site-packages/fsspec/mapping.py in __getitem__(self, key, default)
    142         try:
--> 143             result = self.fs.cat(k)
    144         except self.missing_exceptions:

~/opt/anaconda3/envs/omero-guide-ilastik/lib/python3.7/site-packages/fsspec/asyn.py in wrapper(*args, **kwargs)
    112         self = obj or args[0]
--> 113         return sync(self.loop, func, *args, **kwargs)
    114 

~/opt/anaconda3/envs/omero-guide-ilastik/lib/python3.7/site-packages/fsspec/asyn.py in sync(loop, func, timeout, *args, **kwargs)
     97     elif isinstance(return_result, BaseException):
---> 98         raise return_result
     99     else:

~/opt/anaconda3/envs/omero-guide-ilastik/lib/python3.7/site-packages/fsspec/asyn.py in _runner(event, coro, result, timeout)
     52     try:
---> 53         result[0] = await coro
     54     except Exception as ex:

~/opt/anaconda3/envs/omero-guide-ilastik/lib/python3.7/site-packages/fsspec/asyn.py in _cat(self, path, recursive, on_error, batch_size, **kwargs)
    428             if ex:
--> 429                 raise ex
    430         if (

~/opt/anaconda3/envs/omero-guide-ilastik/lib/python3.7/asyncio/tasks.py in wait_for(fut, timeout, loop)
    413     if timeout is None:
--> 414         return await fut
    415 

~/opt/anaconda3/envs/omero-guide-ilastik/lib/python3.7/site-packages/fsspec/implementations/http.py in _cat_file(self, url, start, end, **kwargs)
    227             out = await r.read()
--> 228             self._raise_not_found_for_status(r, url)
    229         return out

~/opt/anaconda3/envs/omero-guide-ilastik/lib/python3.7/site-packages/fsspec/implementations/http.py in _raise_not_found_for_status(self, response, url)
    209         if response.status == 404:
--> 210             raise FileNotFoundError(url)
    211         response.raise_for_status()

FileNotFoundError: https://minio-dev.openmicroscopy.org/idr/outreach/55500.zarr/.zarray

During handling of the above exception, another exception occurred:

KeyError                                  Traceback (most recent call last)
~/opt/anaconda3/envs/omero-guide-ilastik/lib/python3.7/site-packages/zarr/core.py in _load_metadata_nosync(self)
    228             mkey = _prefix_to_array_key(self._store, self._key_prefix)
--> 229             meta_bytes = self._store[mkey]
    230         except KeyError:

~/opt/anaconda3/envs/omero-guide-ilastik/lib/python3.7/site-packages/zarr/storage.py in __getitem__(self, key)
    716     def __getitem__(self, key):
--> 717         return self._mutable_mapping[key]
    718 

~/opt/anaconda3/envs/omero-guide-ilastik/lib/python3.7/site-packages/fsspec/mapping.py in __getitem__(self, key, default)
    146                 return default
--> 147             raise KeyError(key)
    148         return result

KeyError: '.zarray'

During handling of the above exception, another exception occurred:

ArrayNotFoundError                        Traceback (most recent call last)
/var/folders/t4/z2zdsn015673vzl4lw15_bkm0000gn/T/ipykernel_76570/925900097.py in <module>
     16 for image in values:
     17     filename, file_extension = os.path.splitext(image.getName())
---> 18     input_data = load_from_s3(image)
     19 
     20     # run ilastik headless

/var/folders/t4/z2zdsn015673vzl4lw15_bkm0000gn/T/ipykernel_76570/3922319859.py in load_from_s3(image, resolution)
      4     root = 'idr/outreach/%s.zarr/' % id
      5     # data.shape is (t, c, z, y, x) by convention
----> 6     data = da.from_zarr(endpoint_url + root)
      7     values = data[:]
      8     values = values.swapaxes(1, 2).swapaxes(2, 3).swapaxes(3, 4)

~/opt/anaconda3/envs/omero-guide-ilastik/lib/python3.7/site-packages/dask/array/core.py in from_zarr(url, component, storage_options, chunks, name, inline_array, **kwargs)
   3382             url = os.fspath(url)
   3383         mapper = get_mapper(url, **storage_options)
-> 3384         z = zarr.Array(mapper, read_only=True, path=component, **kwargs)
   3385     else:
   3386         mapper = url

~/opt/anaconda3/envs/omero-guide-ilastik/lib/python3.7/site-packages/zarr/core.py in __init__(self, store, path, read_only, chunk_store, synchronizer, cache_metadata, cache_attrs, partial_decompress, write_empty_chunks, zarr_version)
    201 
    202         # initialize metadata
--> 203         self._load_metadata()
    204 
    205         # initialize attributes

~/opt/anaconda3/envs/omero-guide-ilastik/lib/python3.7/site-packages/zarr/core.py in _load_metadata(self)
    218         """(Re)load metadata from store."""
    219         if self._synchronizer is None:
--> 220             self._load_metadata_nosync()
    221         else:
    222             mkey = _prefix_to_array_key(self._store, self._key_prefix)

~/opt/anaconda3/envs/omero-guide-ilastik/lib/python3.7/site-packages/zarr/core.py in _load_metadata_nosync(self)
    229             meta_bytes = self._store[mkey]
    230         except KeyError:
--> 231             raise ArrayNotFoundError(self._path)
    232         else:
    233 

ArrayNotFoundError: array not found at path %r' ''

```

**Comment by jburel** [comment]
fixed https://github.com/ome/omero-guide-ilastik/pull/36
